rossum_api/api_client.py

```diff
@@ -106,6 +106,7 @@ def __init__(
         timeout: Optional[float] = None,
         n_retries: int = 3,
         retry_backoff_factor: float = 1.0,
+        max_in_flight_requests: int = 4,
```

Comment:
This option should probably be passable from the user-facing classes `ElisAPIClient(Sync)` to allow users to customize it easily but let's merge this as is.